client/src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectFileListing.js

```diff
@@ -60,7 +61,7 @@ const DataFilesProjectFileListing = ({ system, path }) => {
           </Button>
         </div>
       </div>
-      <div styleName="description">{metadata.description}</div>
+      <ReadMore className="shared-workspace__description" text={metadata.description} />
```

Comment:
__Suggestion__

I propose _also_ __or__ instead_ supporting the syntax where `{metadata.description}` is passed as `children`. To access children content, add `children` prop to component. The children prop is supported in several of our components, like `Pill` and `Message` and makes the JSX usage more HTML-y.

```suggestion
      <ReadMore className="shared-workspace__description">{metadata.description}</ReadMore>
```

Reply:
I completely refactored the logic with a resize hook plugin and included children.